api/common/auth.py

```diff
@@ -92,6 +92,18 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def turk_endpoint(f):
```

Comment:
This checks that the endpoint is being hit in `turk` mode (i.e ApiService was in `turk` mode when making this request)